_query-dsl/span/span-or.md

```diff
@@ -0,0 +1,127 @@
+---
+layout: default
+title: Span or
+parent: Span queries
+grand_parent: Query DSL
+nav_order: 70
+---
+
+# Span or query
+
+The `span_or` query combines multiple span queries and matches the union of their spans. It matches if any of its contained span queries match.
```

Comment:
The second sentence here is confusing. Can we rewrite for clarity?

Reply:
Done